wcfsetup/install/files/lib/system/database/table/column/AbstractIntDatabaseTableColumn.class.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace wcf\system\database\table\column;
+
+/**
+ * Abstract implementation of an integer database table column.
+ * 
+ * @author	Matthias Schmidt
+ * @copyright	2001-2019 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\System\Database\Table\Column
+ * @since	5.2
+ */
+abstract class AbstractIntDatabaseTableColumn extends AbstractDatabaseTableColumn implements IAutoIncrementDatabaseTableColumn, ILengthDatabaseTableColumn, IUnsignedDatabaseTableColumn {
+	use TAutoIncrementDatabaseTableColumn;
+	use TLengthDatabaseTableColumn;
```

Comment:
FWIW: The display length of integer columns is deprecated as of MySQL 8 and never made any sense. It does not restrict the values that can be stored. It's just a hint to the MySQL command line client (and possibly other applications) to properly pad the values.

see: https://dev.mysql.com/doc/refman/8.0/en/numeric-type-attributes.html

I'd take this opportunity to remove support for specifying the length of integer fields.